xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3772,6 +3772,7 @@ platform.dashboard.user.displayOnMainPage=Display my dashboard on the wiki home
 platform.dashboard.wiki=Dashboard
 platform.dashboard.wiki.pages=Pages
 platform.dashboard.wiki.tagcloud=Tags
+platform.dashboard.wiki.tagsgadget=Tags for {0}
```

Comment:
To be honest I don't like the key. 

Instead of 'platform.dashboard.wiki.tagsgadget' I would put something like: 
platform.dashboard.wiki.tagcloud.space.title